presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -62,6 +62,7 @@
     private double networkCostWeight = 15;
     private boolean distributedIndexJoinsEnabled;
     private JoinDistributionType joinDistributionType = REPARTITIONED;
+    private boolean distributedJoinsEnabled = true;
```

Comment:
Do we really need to retain `FeatureConfig`-level toggle? With this always being set (`true` or `false`) and having precedence over `joinDistributionType`, `joinDistributionType` should never be consulted, which looks  like not what we want.

Reply:
I don't know how much the featureConfig-level toggle is used, so I don't know whether we need to keep it.  But it looks like you misunderstood how the override of joinDistributionType works.  See comments above.